paper/paper.md

```diff
@@ -0,0 +1,248 @@
+---
+title: 'TransitionInTimeSeries.jl: A performant, extensible and reliable software for
+reproducible detection and prediction of transitions in time series'
+tags:
+ - Julia
+ - nonlinear dynamics
+ - time series analysis
+ - change point detection
+ - resilience loss
+ - critical slowing down
+ - early warning signals
+authors:
+ - name: Jan Swierczek-Jereczek
+   orcid: 0000-0003-2213-0423
+   affiliation: "1, 2"
+ - name: George Datseris
+   orcid: 0000-0003-0872-7098
+   affiliation: 3
+affiliations:
+- name: Department of Earth Physics and Astrophysics, Complutense University of Madrid.
+  index: 1
+- name: Geosciences Institute, CSIC-UCM.
+  index: 2
+- name: Department of Mathematics and Statistics, University of Exeter.
+  index: 3
+date: 13 February 2024
+bibliography: paper.bib
+---
+
+# Summary
+
+Transitions of nonlinear dynamical systems can significantly impact individuals
+and society. Examples of this are ubiquitous and include the onset of cardiac arrhythmia
+[@tse_mechanisms_2016], the deglaciation of Earth about 20,000 years ago [@wolff_changes_2010]
+and the recent price collapse of many cryptocurrencies [@ismail_detecting_2020]. The systems
+displaying such transitions are therefore usually monitored by measuring state variables
+that are believed to be representative of the underlying process. Researchers analyze the
+resulting time series with various methods to detect past transitions and predict
```

Comment:
```suggestion
resulting timeseries with various methods to detect past transitions and predict
```

I've taken it my personal crusade to replace the words "time series" with "timeseries" in the NLD literature.